newsModels/newsModels

```diff
@@ -11,3 +12,19 @@ exports.fetchTopics = (url) => {
         return formattedTopics;
     });
 }
+
+exports.fetchArticles = (url) => {
+
+    return db.query("SELECT * FROM articles").then((articles) => {
```

Comment:
This is a cool solution however we want to avoid doing multiple database queries for comments and articles and sorting these as this is very resource intensive to do on our server.

We can incorporate both the comment count and sort into the query we send off to the database to handle all of this for us.

Please have a look at the count function in PSQL (https://notes.northcoders.com/courses/js-back-end/sql-relationships#aggregate-functions)and the order by section of the notes to see how we can achive this using a database query rarther than a utils function (https://notes.northcoders.com/courses/js-back-end/sql-introduction#order-by)

Reply:
Heartbreaking, but understandable. And doable.